diff --git a/doc/user/meson.build b/doc/user/meson.build
index 39cc0d6e..f02d1b96 100644
--- a/doc/user/meson.build
+++ b/doc/user/meson.build
@@ -19,7 +19,7 @@ if (sphinx_build.found())

 foreach lang : locales
   foreach format : [
-    'html', 'singlehtml', 'latexpdf', 'epub' ]
+    'html', 'singlehtml', 'latex', 'epub' ]

     name = format + '-manual-' + lang
     output =  meson.current_build_dir () / lang / format
diff --git a/inc/audio/engine_windows_mme.h b/inc/audio/engine_windows_mme.h
index 7a00c872..2df5b0da 100644
--- a/inc/audio/engine_windows_mme.h
+++ b/inc/audio/engine_windows_mme.h
@@ -24,7 +24,7 @@
 #ifndef __AUDIO_ENGINE_WINDOWS_MME_H__
 #define __AUDIO_ENGINE_WINDOWS_MME_H__

-#include <Windows.h>
+#include <windows.h>

 #include <gtk/gtk.h>

diff --git a/inc/audio/windows_mmcss.h b/inc/audio/windows_mmcss.h
index 18477fbc..11483c36 100644
--- a/inc/audio/windows_mmcss.h
+++ b/inc/audio/windows_mmcss.h
@@ -42,7 +42,7 @@
 #ifndef __AUDIO_WINDOWS_MMCSS_H__
 #define __AUDIO_WINDOWS_MMCSS_H__

-#include <Windows.h>
+#include <windows.h>

 typedef enum AVRT_PRIORITY
 {
diff --git a/inc/audio/windows_mme_device.h b/inc/audio/windows_mme_device.h
index 86655cb9..baeee19e 100644
--- a/inc/audio/windows_mme_device.h
+++ b/inc/audio/windows_mme_device.h
@@ -44,7 +44,7 @@

 #include <stdint.h>

-#include <Windows.h>
+#include <windows.h>

 #include "zix/ring.h"

diff --git a/inc/utils/windows_errors.h b/inc/utils/windows_errors.h
index d36ec9db..d0d66935 100644
--- a/inc/utils/windows_errors.h
+++ b/inc/utils/windows_errors.h
@@ -22,7 +22,7 @@
 #ifndef __UTILS_WINDOWS_ERRORS_H__
 #define __UTILS_WINDOWS_ERRORS_H__

-#include <Windows.h>
+#include <windows.h>

 void
 windows_errors_get_last_error_str (
diff --git a/meson.build b/meson.build
index 0514ab8f..1fef90c6 100644
--- a/meson.build
+++ b/meson.build
@@ -580,6 +580,11 @@ if sdl_dep.found ()
   cdata.set('HAVE_SDL', 1)
 endif

+zstd_dep = dependency ('libzstd', required: false)
+if not zstd_dep.found ()
+  zstd_dep = cc.find_library ('zstd')
+endif
+
 # TODO add Cantarell font as dependency

 zrythm_deps = [
@@ -603,7 +608,7 @@ zrythm_deps = [
   rtaudio_dep,
   rtmidi_dep,
   sdl_dep,
-  dependency('libzstd'),
+  zstd_dep,
   dependency('threads'),
   lilv_dep,
   dependency(
